index.html

```diff
@@ -452,14 +440,9 @@ <h2>
           Let <var>p</var> be a new <a>Promise</a> object.
         </li>
         <li>
-          Run the following steps <a>in parallel</a>:
-          <ol>
-            <li>
-              If <a>obtain permission</a> steps return <code>false</code>,
-              then reject <var>p</var> with <code>"<a>NotAllowedError</a>"</code>
-              <a>DOMException</a>, and abort these steps.
-            </li>
-          </ol>
+          If <a>obtain permission</a> steps return <code>false</code>,
+          then reject <var>p</var> with <code>"<a>NotAllowedError</a>"</code>
```

Comment:
nit: while here, "with NotAllowedError DOMException" -> "with _the_ NotAllowedError DOMException".

Reply:
... and, of course, nitpicking at my own nit: _a_ NotAllowedError DOMException

Reply:
fixed